clients/pricing-client/src/openapi.d.ts

```diff
@@ -2038,6 +2031,13 @@ declare namespace Components {
         }
         export interface PriceGetAg {
             category: string;
+            markup_pricing_model?: "per_unit" | "tiered_volume" | "tiered_flatfee";
+            type?: "base_price" | "work_price";
```

Comment:
I would create a specific component schema in the spec for this so we can easily reuse here + have access to the specific type easily, what you think?